integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -155,18 +161,22 @@ def get_low_hix_value_context(self):
         :return: Dictionary containing the context for pages with low hix value
         :rtype: dict
         """
-        if settings.TEXTLAB_API_ENABLED and self.request.region.hix_enabled:
-            translations_under_hix_threshold = PageTranslation.objects.filter(
-                language__slug__in=settings.TEXTLAB_API_LANGUAGES,
-                id__in=self.latest_version_ids,
-                page__hix_ignore=False,
-                hix_score__lt=settings.HIX_REQUIRED_FOR_MT,
-            )
+        if not settings.TEXTLAB_API_ENABLED or not self.request.region.hix_enabled:
+            return {}
+
+        translations_under_hix_threshold = PageTranslation.objects.filter(
+            language__slug__in=settings.TEXTLAB_API_LANGUAGES,
+            page__id__in=Subquery(
+                self.request.region.non_archived_pages.filter(hix_ignore=False).values(
+                    "pk"
+                )
+            ),
+            hix_score__lt=settings.HIX_REQUIRED_FOR_MT,
+        ).distinct("page_id", "language_id")
```

Comment:
```suggestion
            id__in=self.latest_version_ids,
            page__hix_ignore=False,
            hix_score__lt=settings.HIX_REQUIRED_FOR_MT,
        )
```
Small mistake in the conflict resolution